package.json

```diff
@@ -30,9 +30,11 @@
   ],
   "dependencies": {
     "carbon-icons": "^6.0.4",
+    "flatpickr": "^2.6.3",
     "lodash.debounce": "^4.0.8"
   },
   "devDependencies": {
+    "@ibma/karma-ibma": "^2.2.0",
```

Comment:
It looks we removed this from our package.json in [`master` branch](https://github.com/carbon-design-system/carbon-components/blob/master/package.json#L35)

Right now, the current failing builds in Travis are because npm cannot find `@ibm/karma-ibma`...I've also excluded this [devDependency from my PR](https://github.com/carbon-design-system/carbon-components/pull/140) and Travis is passing. 

I'll keep digging to see if there are any other factors at play.